pkg/adapter/adapter.go

```diff
@@ -126,6 +126,12 @@ func (adp *LocalAdapter) Convert(ctx context.Context, source string) error {
 	adp.content.GcMutex.RLock()
 	defer adp.content.GcMutex.RUnlock()
 	if _, err = adp.cvt.Convert(ctx, source, target, cacheRef); err != nil {
+		if errdefs.NeedsRetryWithoutCache(err) && cacheRef != "" {
+			logrus.Infof("retry convert without cache: %s", cacheRef)
```

Comment:
Sorry for the late comment, can we tell the user what's happened here? How about `"inconsistent layer format with the cache, retry conversion without cache: %s"`.

Reply:
sure！